### DISCUSSION TITLE
Completion doesn't work in injections.

### DISCUSSION THREAD

**Discussion by vanarok** [helix-editor/helix#11068] [Q&A] [answered]
For some reason, completion doesn't work in tag attributes, but it works inside the script tag, and I don't understand why.
![image](https://github.com/helix-editor/helix/assets/6105977/860f4a0e-22c0-44ad-97b6-4495dd620364)

**Comment by the-mikedavis** [comment, marked as answer]
Completion is provided by the language server and doesn't interact with tree-sitter and how we parse injections. It's up to the language server to figure out what completions to offer in each area of the document.
